terraform/production.tf

```diff
@@ -81,7 +83,7 @@ resource "aws_opsworks_instance" "apr-production-backend" {
   layer_ids = [
     "${aws_opsworks_custom_layer.apr-backend.id}",
   ]
-  instance_type = "t2.small"
+  instance_type = "t2.medium"
```

Comment:
do we still need it to be medium? or since we found the compile issue, small would be still enough?